yagmail/message.py

```diff
@@ -230,6 +230,11 @@ def get_mime_object(is_marked_up, content_string, encoding):
             content_object["main_type"] = "application"
             content_object["sub_type"] = "octet-stream"
 
+    # Fixed the problem in issue: https://github.com/kootenpv/yagmail/issues/242
+    if content_object["main_type"] == 'text' and content_object["sub_type"] == 'plain':
+        content_object["main_type"] = 'application'
+        content_object["sub_type"] = 'octet-stream'
+
```

Comment:
Thanks for your contribution! 

Maybe it means we should never define the codebase then to use `text/plain`? 

I'm kinda confused how `application/octet-stream` would be good when wanting to send text?

Reply:
I just found inspiration here: https://stackoverflow.com/questions/9541837/attach-a-txt-file-in-python-smtplib/48417682#48417682

Reply:
And it work well on alimail(https://qiye.aliyun.com/)，qqmail(https://mail.qq.com), 126mail(https://mail.126.com) and gmail when sending  .txt attachment file

Reply:
I just upload a fork of pypi: https://pypi.org/project/yagmail2/